src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -293,68 +294,70 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
       round-slider {
         max-width: 200px;
         --round-slider-path-width: 35px;
-        --round-slider-path-color: var(--disabled-text-color);
+        --round-slider-path-color: var(--primary-background-color);
```

Comment:
Is this right? Shouldn't it be the gray color? At least that's what it has been I thought

Reply:
Should definitely not be a text color, I think this was used before the round slider change